### PR TITLE
Added Variable Information

### PR DESCRIPTION
**Pull request by fizarasool27**

### CODE REVIEW

**Comment by akshitgrover** [comment]
Hey @fizarasool27
Please put examples in the example folder.
Try and elaborate these topics more.

**Comment by fizarasool27** [comment]
@akshitgrover 
Examples are to explain the topic in a better way. If we put examples in another folder, it will get confusing as users will not be able to connect examples with topic.
Which section do you suggest needs elaboration?

**Comment by akshitgrover** [comment]
@fizarasool27
Leave them as it is in the md file, We need executables as well in the examples folder.
Basically, python code, name the file as variables.py in the examples folder

File: basics/variables.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+>>> foo
+NameError: name 'foo' is not defined
+```
+* You can also take the value of the variable from the user input.
+```
+>>> foo = input("Enter a number: ")
+Enter a number: 7
+>>> print(foo)
+7
+```
```
Comment (OctaGRIMME) [inline basics/variables.md:65]: As we have discussed , can you please provide the links

Comment (akshitgrover) [reply basics/variables.md:65]: Also, Remove these ">>>" python shell cursors.

**Comment by akshitgrover** [comment]
Hey @fizarasool27 
I went through the file, It is good to go, I don't think any more changes is required.
Also, Since it is just variables section, I don't think any explicit examples are required in the folder.
Good to merge :+1:

**Pull request merged**